backend/src/contracts/tests.rs

```diff
@@ -1,24 +1,69 @@
-mod test {
+use std::{sync::Arc, time::Duration};
+
+use ethers::{
+    prelude::{ContractFactory, SignerMiddleware},
+    providers::{Http, Provider},
+    signers::{LocalWallet, Signer},
+    types::{H160, U256},
+};
+
+use crate::contracts::generated::mock_erc20::{MockERC20, MOCKERC20_ABI, MOCKERC20_BYTECODE};
+
+pub async fn initialize_anvil(
```

Comment:
Can you specify in the comments what's happening here? What are you initializing at?

Reply:
I will update it while fixing error on github action

Reply:
Updated at 8502144c